screen-orientation/ios/Plugin.xcworkspace/contents.xcworkspacedata

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
this file should be in .gitignore

Reply:
This file is in other plugins' `ios/Plugin.xcworkspace` folders. We don't have a `.gitignore` file for `ios/` folders, want me to add this to the main `.gitignore`?

Reply:
yes please. .xcworkspacedata is meant for local configuration